### PR TITLE
docs: add evalation order of env to readme

### PR DESCRIPTION
**Pull request by yk634**
## Context

For requests of https://github.com/screwdriver-cd/sd-local/pull/74#issuecomment-951476434.

## Objective

I added evalation order of environment variable for a build to README.

## References

https://github.com/screwdriver-cd/sd-local/pull/74

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,11 @@ Flags:
 Global Flags:
   -v, --verbose   verbose output.
 ```
+* There are some ways to set environment variables for a build. If the same key is set in more than one way, the priority is as follows:
+  1. `--env` or `-e` Flag
+  1. `--env-file` Flag
+  1. environment of screwdriver.yaml
+  1. defaultValue (e.g.: `SD_TOKEN`, `SD_API_URL`)
```
Comment (tk3fftk) [inline README.md:67]: camel case?
```suggestion
  1. default value (e.g.: `SD_TOKEN`, `SD_API_URL`)
```

**Review by wahapo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -60,6 +60,11 @@ Flags:
 Global Flags:
   -v, --verbose   verbose output.
 ```
+* There are some ways to set environment variables for a build. If the same key is set in more than one way, the priority is as follows:
+  1. `--env` or `-e` Flag
+  1. `--env-file` Flag
+  1. environment of screwdriver.yaml
```
Comment (tk3fftk) [inline README.md:66]: for more accuracy
```suggestion
  1. `environment` in a screwdriver.yaml
```

File: README.md (hunk 1 above)
Comment (tk3fftk) [reply README.md:67]: or did you mean `defaultEnv` ? ( refer to https://github.com/screwdriver-cd/sd-local/pull/74#issue-1033432302 )
https://github.com/screwdriver-cd/sd-local/blob/fa1ece6995a8d836622ec8b92819a06c6ccda06a/launch/launch.go#L140-L146

Comment (yk634) [reply README.md]: I fixed.

File: README.md (hunk 2 above)
Comment (yk634) [reply README.md]: I fixed.

**Review by tk3fftk — APPROVED** [review]
:+1:

**Review by itleigns — APPROVED** [review]

**Review by jithine — APPROVED** [review]

**Pull request merged**